cmd/lakefs/cmd/run.go

```diff
@@ -53,6 +53,9 @@ var runCmd = &cobra.Command{
 		// validate service names and turn on the right flags
 		dbParams := cfg.GetDatabaseParams()
 
+		if err := db.ValidateSchemaUpToDate(conf.GetDatabaseParams()); err != nil {
```

Comment:
1. `conf` and probably it is not you - but all the commands in lakefs should use `cfg`
2. you don't need to call cfg.GetDatabaseParams() you have them in `dbParams` from line 54